### PR TITLE
Add master annotation

### PR DESCRIPTION
**Pull request by komalsrathi**
Per slack conversation with @mkoptyra, he has requested for a combined annotation file that has information used by availability plots and Oncoplots.

### CODE REVIEW

**Comment by komalsrathi** [comment]
@jharenza any chance of reviewing and getting this merged this week? Thank you!

File: analyses/master_annotation/01-master_annotation.R [hunk 1]
```diff
@@ -0,0 +1,143 @@
+suppressPackageStartupMessages({
+  library(tidyverse)
+})
+
+# set directories
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+data_dir <- file.path(root_dir, "data")
+analyses_dir <- file.path(root_dir, "analyses", "master_annotation")
+output_dir <- file.path(analyses_dir, "results")
```
Comment (jharenza) [inline analyses/master_annotation/01-master_annotation.R:9]:
```suggestion
tmb_dir <- file.path(root_dir, "analyses", "tmb-calculation")
output_dir <- file.path(analyses_dir, "results")
```

File: analyses/master_annotation/01-master_annotation.R [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+# add new Diagnosis to annotation 
+annot <- annot %>%
+  left_join(diagnosis_who_grade_map)
+
+# for tumor-only samples use WGS_tumor_only under experimental_strategy
+annot <- annot %>%
+  mutate(experimental_strategy = ifelse(Kids_First_Biospecimen_ID %in% wgs_tumor_only_ids, "WGS_tumor_only", experimental_strategy))
+
+# pull TMB (for T/N paired samples)
+tmb_paired_output <- read_tsv("analyses/tmb-calculation/results/wgs_paired/snv-mutation-tmb-coding.tsv") %>%
```
Comment (jharenza) [inline analyses/master_annotation/01-master_annotation.R:60]:
```suggestion
tmb_paired_output <- read_tsv(file.path(tmb_dir, "results/wgs_paired/snv-mutation-tmb-coding.tsv")) %>%
```

File: analyses/master_annotation/01-master_annotation.R [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  dplyr::rename("TMB" = "tmb") %>%
+  dplyr::select(Tumor_Sample_Barcode, TMB) %>%
+  filter(Tumor_Sample_Barcode %in% wgs_ids) %>%
+  inner_join(annot, by = c("Tumor_Sample_Barcode" = "Kids_First_Biospecimen_ID")) %>%
+  dplyr::select(sample_id, TMB) %>%
+  unique()
+
+# pull TMB (for tumor-only samples)
+tmb_tumor_only_output <- read_tsv("analyses/tmb-calculation/results/wgs_tumor_only/snv-mutation-tmb-coding.tsv") %>%
```
Comment (jharenza) [inline analyses/master_annotation/01-master_annotation.R:69]:
```suggestion
tmb_tumor_only_output <- read_tsv(file.path(tmb_dir, "results/wgs_tumor_only/snv-mutation-tmb-coding.tsv")) %>%
```

**Review by jharenza — CHANGES_REQUESTED** [review]
The directory for TMB needs to be updated here, as it is searching within the annotation dir, so I made those suggestions. Following those, this runs and reproduces. 

Also, suggest to add the instruction `Rscript --vanilla 01-master_annotation.R` to the readme or add a run script.

**Review by jharenza — APPROVED** [review]

**Pull request merged**